apis/v1alpha1/gateway_types.go

```diff
@@ -234,6 +234,39 @@ type Listener struct {
 	//
 	// Support: Core
 	Routes RouteBindingSelector `json:"routes"`
+
+	// RouteTo defines how to route requests directly to a backend without
+	// creating an xRoute resource.
+	//
+	// If the referent cannot be found, the controller should raise
+	// the "ResolvedRefs" condition on the Gateway with the "DegradedListeners"
+	// reason. The gateway status should be updated with a condition
+	// that describes the error more specifically.
+	//
+	// Support: Extended
```

Comment:
I think if we base support off of protocol, we could consider say this has core support for core protocols (as defined slightly lower in this file).